test/integration/collector-remote-method.tap.js

```diff
@@ -13,6 +13,8 @@ const url = require('url')
 const collector = require('../lib/fake-collector')
 const RemoteMethod = require('../../lib/collector/remote-method')
 
+const SSL_HOST = 'localhost'
```

Comment:
thoughts on moving this `SSL_HOST` to a test constant that can be referenced in unit, integration, versioned tests?

Reply:
Hopefully would never be used by the unit tests haha (yes they already do bad thing slike call example.com).

I considered it but I decided the tradeoff wasn't currently worth it. But I'm not strong against.

Just seemed like plumbing the constant into a central location that potentially adds another file dependency wasn't really worth it for how few places were using. I guess that SSL helper could return the constant at some point and that wouldn't be too bad. I just like to be wary of adding new problems to avoid duplication (but yes don't want to update 1000 files when you have to make a change either).

Now that I'm done thinking out loud, I'm fine either way.

Reply:
yea my thought here was this PR was quite involved to change the host in a cert and its corresponding tests that use the cert

Reply:
i made these changes but don't have perms to push to your fork, so ill just open a separate PR after I merge this